src/tests/JIT/Directed/forceinlining/AttributeConflict.ilproj

```diff
@@ -9,6 +9,8 @@
   </PropertyGroup>
   <PropertyGroup>
     <JitOptimizationSensitive>true</JitOptimizationSensitive>
+    <!-- Interpreter does not support Inlining -->
```

Comment:
Instead of this, should we disable all tests marked with `JitOptimizationSensitive` when running w/ interpreter?

Reply:
@jkotas, in the long term, we may want to do that, but at the moment I'd like to be more particular.